svc-api/handle/telemetry_test.go

```diff
@@ -42,6 +42,29 @@ func testTelemetryService(req teleproto.TelemetryRequest) (*teleproto.TelemetryR
 	return response, nil
 }
 
+func mockUpdateTriger(req teleproto.TelemetryRequest) (*teleproto.TelemetryResponse, error) {
```

Comment:
thanks for actioning this. I suggested using anon funcs to Jeevan. That should make these more manageable.